src/test/regress/expected/typed_streams.out

```diff
@@ -4,11 +4,10 @@ SET search_path TO test_typed_streams,public;
 CREATE STREAM s0 (x integer, y integer);
 DROP TABLE s0;
 ERROR:  "s0" is not a table
-HINT:  Use DROP STREAM to remove a stream.
+HINT:  Use DROP FOREIGN TABLE to remove a foreign table.
```

Comment:
We should probably talk more about the best balance to strike here. Whatever we end up doing will trade aestheticism for internal code complexity. Adding another relkind essentially means that we have to check for `RELKIND_STREAM` wherever `RELKIND_FOREIGN_TABLE` is referenced.